src/status_im2/contexts/shell/activity_center/drawer/view.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im2.contexts.shell.activity-center.drawer.view
+  (:require [utils.re-frame :as rf]
+            [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]))
+
+(defn options-bottom-sheet-content
```

Comment:
`options-bottom-sheet-content` can be renamed to just `options` as it's an inside `drawer` ns and it's self-explanatory.

Reply:
Done